charts/elastic-cloud/README.md

```diff
@@ -0,0 +1,49 @@
+# Install Operator
+## Add Elasticsearch Helm Repositories
+```
+helm repo add elastic https://helm.elastic.co
+helm repo update
+```
+
+## Install operator and CRDs (Not Cluster-wide)
+#### Due to us needing the CRDs at the time of template installation we MUST install these items manually.
+```
+helm install elastic-operator-crds elastic/eck-operator-crds
+```
+```
+#### Install CRDs outside of operator so cluster remains even if operator is deleted
+#### We specify a namespace here, but you can opt to install it clusterwide
+
+helm install elastic-operator elastic/eck-operator -n <Namespace> \
+    --set=installCRDs=false \
+    --set=managedNamespaces='{<Namespace>}' \
+    --set=createClusterScopedResources=false \
+    --set=webhook.enabled=false \
+    --set=config.validateStorageClass=false
+```
+
+## Uninstall Operator and CRDs
+```
+helm uninstall elastic-operator-crds -n <Namespace>
+helm uninstall elastic-operator -n <Namespace>
+```
+
+## Install example deloyment (ES (3 master/ 3 data) Kibana (1 instance))
+#### This example values file also includes support for ingress and snapshot backups, but you'll have to enable them with your specific values
+```
+helm install elastic-cloud --namespace <Namespace> -f examples/multi-node.yaml .
+```
+
+## Register snapshot repository S3
```

Comment:
We should automate the snapshot repo creation with Helm's `post-install` hook to execute this 
 curl command. https://helm.sh/docs/topics/charts_hooks/

Once we get the hook working, we can also supply the bucket, basepath and region in `Values.backups.aws`

```yaml
backups:
  aws:
    ...
    region: us-east-1
    bucket: myapp-backups
    base_path: production/
```
It'll be similar to your [snapshotter](https://github.com/rhoai/helm-charts/pull/1/files#diff-80f8b0b365b0c56be702b4a0d7dec678d7e91fb030102e2dc97d001e51c1dbc5), but include helm hook annotations.

Reply:
Like we talked about here this isn't going to work as a post-install hook without more complex logic to wait for es cluster to be ready.  We'll leave as a future improvement.